README.rst

```diff
@@ -4,14 +4,111 @@
 .. image:: https://travis-ci.org/camptocamp/marabunta.svg?branch=master
     :target: https://travis-ci.org/camptocamp/marabunta
 
-Marabunta is a name given to the migration of the legionary ants or to the ants
-themselves. Restless, they eat and digest everything in their way.
+Marabunta is used to provide an easy way to provide updates to Odoo which can be created fast and run easily. It also allows to differentiate between different environment to provide for instance demodata.
 
-This tool aims to run migrations for Odoo versions as efficiently as a
-Marabunta migration.
 
-It loads migration instructions from a YAML file and run the operations if
-required.
+Usage
+=====
+After installing marabunta will be available as a console command. To run properly it requires a migration file, which defines what should be updated/executed, and of course stuff like database pw. Most of these also can also be provided by an envvar and will be discussed in the options section.
+
+It is very much recomended to configure it, so that marabunta is ran automatically if odoo is started.
```

Comment:
At each run, marabunta will verify the versions from the migration file and process new ones. It is very much recommended to ...

Reply:
"also can also" -> "can also"

Reply:
s/recomended/recommended

Reply:
s/ran/run